pkg/cluster/mpi/statefulset.go

```diff
@@ -26,16 +27,18 @@ var (
 	workerProbe = &corev1.Probe{
 		Handler: corev1.Handler{
 			TCPSocket: &corev1.TCPSocketAction{
-				Port: intstr.FromInt(int(sshdPort)),
+				Port: intstr.FromInt(sshdPort),
 			},
 		},
 	}
 	workerCommand = []string{
-		"/usr/sbin/sshd",
-		"-De",
+		launchScriptPath,
 	}
 )
 
+// Key of the shared Secret object that contains client-side SSH public key
+const publicKeyField = "ssh-publickey"
```

Comment:
I am suprirsed this isn't a constant somewhere in the k8s lib, I don't think we set it manually

Reply:
Agree. There may be.

Reply:
`corev1.SSHAuthPrivateKey`